src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs

```diff
@@ -158,6 +159,27 @@ internal class RPackageManagerViewModel : BindableBase, IRPackageManagerViewMode
             DispatchOnMainThread(() => InstallAsync(package));
         }
 
+        private async Task InstallAsync(IRPackageViewModel package) {
+            _coreShell.AssertIsOnMainThread();
+            if (_selectedTab == SelectedTab.InstalledPackages) {
+                IsLoading = true;
+            }
+
+            try {
+                var libPath = await GetLibPath();
+                await _packageManager.InstallPackageAsync(package.Name, libPath);
+            } catch (RPackageManagerException ex) {
+                AddErrorMessage(ex.Message);
+            }
+
+            await ReloadInstalledAndLoadedPackagesAsync();
+            if (_selectedTab == SelectedTab.InstalledPackages) {
+                IsLoading = false;
+                ReplaceItems(_installedPackages);
+            }
+            package.IsChanging = false;
```

Comment:
Should it be set to true here? Or is it set elsewhere? I think we may consider something like using(var x = package.BeginChange()) so we won't have to track changing state set/clear